src/effects/firework_rocket.c

```diff
@@ -6,6 +6,64 @@ void firework_rocket_render(EffectInstance* effect);
 void firework_rocket_update(EffectInstance* effect);
 void firework_rocket_appendGfx(void* effect);
 
+typedef struct FireworkRocketUnk {
+    /* 0x00 */ Color_RGB8* unk_00;
+    /* 0x04 */ s32 num;
+} FireworkRocketUnk; // size = 0x8
+
+extern Gfx D_09000080_3E53D0[];
+extern Gfx D_09000158_3E54A8[];
+
+Gfx* D_E010ACC0[] = { D_09000158_3E54A8, D_09000158_3E54A8, D_09000158_3E54A8 };
+Gfx* D_E010ACCC[] = { D_09000080_3E53D0, D_09000080_3E53D0, D_09000080_3E53D0 };
+
+Color_RGB8 D_E010ACD8[] = {
+    {36, 0, 93}, {54, 216, 74}, {67, 0, 74}, {82, 220, 45}, {89, 0, 45}, {60, 190, 45}, {21, 192, 74}, {28, 171, 45}, 
```

Comment:
We should split these images (?) now if possible

Reply:
they're not images, they're rgb values used in the graphics func